OWML.ModHelper.Events/HarmonyHelper.cs

```diff
@@ -9,11 +9,34 @@ public class HarmonyHelper : IHarmonyHelper
     {
         private readonly IModLogger _logger;
         private readonly IModConsole _console;
+        private readonly HarmonyInstance _harmony;
 
         public HarmonyHelper(IModLogger logger, IModConsole console)
         {
             _logger = logger;
             _console = console;
+            _harmony = CreateInstance();
+        }
+
+        private HarmonyInstance CreateInstance()
+        {
+            HarmonyInstance harmony;
+            try
+            {
+                _logger.Log("Creating harmony instance");
+                harmony = HarmonyInstance.Create("com.alek.owml");
```

Comment:
make HarmonyHelper unique per mod, and:
harmony = HarmonyInstance.Create("com.owml." + manifest.UniqueName);
?